cfgov/cfgov/settings/base.py

```diff
@@ -377,27 +377,18 @@
     "search.elasticsearch_helpers.WagtailSignalProcessor"
 )
 
-# S3 Configuration
-# https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings
-AWS_LOCATION = "f"  # A path prefix that will be prepended to all uploads
-AWS_QUERYSTRING_AUTH = False  # do not add auth-related query params to URL
-AWS_S3_FILE_OVERWRITE = False
-AWS_S3_SECURE_URLS = True  # True = use https; False = use http
```

Comment:
The `AWS_S3_SECURE_URLS` option was removed from django-storages [in release 1.13](https://github.com/jschneier/django-storages/blob/master/CHANGELOG.rst#113-2022-08-05).